windows/web/slimevr_web_installer.nsi

```diff
@@ -108,7 +108,7 @@ Function cleanInstDir
     Delete "$INSTDIR\uninstall.exe"
     Delete "$INSTDIR\run.bat"
     Delete "$INSTDIR\run.ico"
-    Delete "$INSTDIR\slimevr.jar"
+    Delete "$INSTDIR\slimevr*"
```

Comment:
Is this safe to do? This works for now, but if anyone adds something starting with `slimevr` then it may cause unintended consequences

Reply:
This will delete only files starting with `slimevr`, but why would anyone add such files? Also note that this is uninstallation sequence.

Reply:
Okay, I'm fine with it, I just wanted to at least mention it 👍